src/DIRAC/TransformationSystem/Client/TransformationFilesStatus.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module contains constants and lists for the possible Transformation files states.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+__RCSID__ = "$Id$"
+
+#:
+UNUSED = 'Unused'
+#:
+ASSIGNED = 'Assigned'
+#:
+PROCESSED = 'Processed'
+#:
+PROBLEMATIC = 'Problematic'
+#:
+MAX_RESET = 'MaxReset'
+#:
+MISSING_IN_FC = 'MissingInFC'
+#:
+PROB_IN_FC = 'ProbInFC'
+#:
+REMOVED = 'Removed'
+
+#: below ones are for inherited transformations
```

Comment:
```suggestion
# below ones are for inherited transformations
#:
```

All the entries are sourced alphabetically, so at the moment this string shows up at the bottom of the documentation page. We can either just make it be only in the source code documentation (like the proposal), or we would need to disable sorting for some modules.